### PR TITLE
v0.7.8 - bug with messages fixed

### PR DESCRIPTION
**Pull request by Sainomori**

### CODE REVIEW

**Review by marcserrat — APPROVED** [review]

**Comment by Sainomori** [comment]
image
image
image
image

File: connect_telegram_ext/models.py [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
-from typing import Optional
+
+@dataclass
+class Event:
+    name: str
+    statuses: list
+    title: str
+    path: str
+    # TODO: replace any with proper type
+    message_callback: Callable[[any, ConnectClient, dict], str]
```
Comment (ffaraone) [inline connect_telegram_ext/models.py]: This `any` is not `typing.Any`, it is the built-in `any` function.

File: docker-compose.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ version: '3'
 services:
   telegram_notify_dev:
     container_name: telegram_notify_dev
-    image: cloudblueconnect/connect-extension-runner:26.33
+    image: cloudblueconnect/connect-extension-runner:26.36
```
Comment (marcserrat) [inline docker-compose.yml]: Why you don't move to new schema using a Dockerfile?? As a sample shall look like bootstraped with our CLI

**Review by marcserrat — APPROVED** [review]

File: connect_telegram_ext/messages.py [hunk 3]
```diff
@@ -19,37 +19,137 @@
...
+    if not brand_id:
+        brand_id = 'BR-000'
+    brand = client.branding('brand').get(params={'id': brand_id})
+    domain = brand['portals'][list(brand['portals'])[0]]['domain']
+    domain = f"https://{domain}/"
+    return f"{domain}{path}/{object_id}"
+
+
+def default_message_callback(event: Event, client: ConnectClient, request: dict) -> str:
+    message_template = getattr(
```
Comment (ffaraone) [inline connect_telegram_ext/messages.py:125]:
```
message_template = getattr(
    Messages,
    f"{event.name.upper()}_{request[event.status_filed]}",
    getattr(
        Messages,
        f"{event.name.upper()}",
        Messages.DEFAULT_MESSAGE,
    )
)
```

**Review by ffaraone — CHANGES_REQUESTED** [review]
Can you also rename `connect_telegram_ext.models.TestMessagePayload` to a name that don't start with `Test` so we remove this annoying warning that pytest emits ?
Thanks!

**Review by marcserrat — APPROVED** [review]

**Review by ffaraone — APPROVED** [review]

**Pull request merged**